paper.tex

```diff
@@ -1929,8 +1983,43 @@ \section{Conclusion and Future Work}
 Kinkaku-ji. More extensive user testing at the actual site could have helped us
 catch bugs earlier and lead to a more polished final product.
 
-Despite these issues our team is happy with the product we delivered to
-Kyoto~VR. Selling ARuko on app stores was not within the scope of our academic
+While not a disaster, we had some initial trouble getting started with Unity.
+Different versions of Unity do not play well with each other, and it often runs
```

Comment:
change to "Different versions of Unity do not play well with each other. As a result, we often ran into bugs on older hardware..."

Reply:
Fixed